docs/querying/caching.md

```diff
@@ -82,17 +82,3 @@ Note that the task executor processes only support caches that keep their data l
 This restriction exists because the cache stores results at the level of intermediate partial segments generated by the
 ingestion tasks. These intermediate partial segments will not necessarily be identical across task replicas, so
 remote cache types such as `memcached` will be ignored by task executor processes.
-
-## Unsupported queries
```

Comment:
Hi @sthetland  what was the reason for removing this para in this PR? was it accidental?